scripts/fixup_admin_v1_keywords.py

```diff
@@ -17,7 +17,12 @@
 
 import argparse
 import os
-import libcst as cst
+
+try:
+    import libcst as cst
+except ImportError as exception:
+    raise ImportError(exception.__class__.__name__ + ": " + exception.msg)
```

Comment:
Pointless to raise another `ImportError` without extra info.  Instead, please add the message specified by @crwilcox, suggesting that the user run  `python -m pip install "libcst >= 0.2.5"`.